src/Ginger.php

```diff
@@ -44,4 +50,35 @@ public static function createClient($apiKey)
             )
         );
     }
+
+    /**
+     * Method restores dashes in Ginger API key in order to validate UUID.
+     *
+     * @param string $apiKey
+     * @return string UUID
+     */
+    public static function apiKeyToUuid($apiKey)
```

Comment:
This function looks nasty... There must be a cleaner way to do this!

Maybe you can use the UUID lib at https://github.com/ramsey/uuid. Or else use a `preg_replace`?

```php
$uuid_hex = '663138387bca44eb9885b650d1f0026c';
$uuid = preg_replace('/(\w{8})(\w{4})(\w{4})(\w{4})(\w{12})/', '$1-$2-$3-$4-$5', $uuid_hex);
echo $uuid;
```

(From https://www.linkedin.com/pulse/20141204164718-2397294-convert-uuid-without-dashes-to-uuid-with-dashes?articleId=5946310918399873024)